deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: wildfly-operator
       containers:
         - name: wildfly-operator
-          image: quay.io/wildfly/wildfly-operator:latest
+          image: quay.io/jmesnil/wildfly-operator:latest
```

Comment:
is this correct or leftover from development?

Reply:
oops, that's a leftover

Reply:
can you please look at others `https://github.com/jmesnil` occurrences too?

Reply:
still here after recent updates, according to your answer I guess it should be fixed before merging

Reply:
fixed